### ISSUE TITLE
 Standard names: *Global average sea level change due to water mass budget*

### ISSUE THREAD

**Issue by abiardeau** [cf-convention/discuss#314]
Hello,
I am Aurore BIARDEAU from Mercator Ocean international, working for Copernicus Marine service.
Date : 13-05-2024

We are in the process of implementing a new variable in the Copernicus Marine service and we would be happy to have your feedback on our proposal :

standard_name : `global_average_sea_level_change_due_to_water_mass_budget `
definition : Global average mass volume sea level change is caused by water mass balance (evaporation – precipitation + runoff). This in turn results in a change in volume of the world ocean. Zero sea level change is an arbitrary level. Because global average sea level change quantifies the change in volume of the world ocean, it is not calculated necessarily by considering local changes in mean sea level

I can provide more information from my expert colleagues if needed,
Kind regards,
Aurore

**Comment by JonathanGregory** [comment]
Dear Aurore @abiardeau

By "due to water mass budget" do you mean "due to change in ocean mass" i.e. barystatic sea level change?

Best wishes

Jonathan

**Comment by abiardeau** [comment]
Yes, that is what we mean.
 Would _barystatic be more appropriate ? Or _due_to_change_in_ocean_mass ?

Thanks, Aurore

**Comment by JonathanGregory** [comment]
I think `due_to_change_in_ocean_mass` is better for a CF standard name, because it's self-explanatory. The description should mention that this quantity is called "barystatic sea level change" in scientific literature, but "barystatic" is not (yet!) a generally well-known word.

**Comment by efisher008** [comment]
Dear Aurore,

Thank you for your proposal, I have added the term to the CF editor.
If you are happy with @JonathanGregory's suggestion of `global_average_sea_level_change_due_to_change_in_ocean_mass`, I can make this change in the editor. Perhaps we can also add the text _"This quantity is also called "barystatic sea level change"."_ 

Would a definition of `barystatic` in the CF phrasebank be appropriate @JonathanGregory? Something along the lines of _" Barystatic refers to the effect of addition or removal, to or from the ocean, of water mass that formerly resided within the land area (as land water storage or land ice) or in the atmosphere (which contains a relatively tiny mass of water)"_, following the description of the term _barystatic sea level rise_ in the following [paper](https://doi.org/10.1007/s10712-019-09525-z). 

Best regards,
Ellie

**Comment by JonathanGregory** [comment]
I agree that it would be useful to mention "barystatic" in the definition of `global_average_sea_level_change_due_to_change_in_ocean_mass`. "Barystatic" is a word that was invented for the phrase "barystatic sea level rise", so it would be better to quote to the whole phrase, I think:

> This quantity is also called "barystatic sea level rise" or "barystatic sea level change". It is the part of global-mean sea-level rise which is due to the addition to the ocean of water mass that formerly resided within the land area (as land water storage or land ice) or in the atmosphere (which contains a relatively tiny mass of water).

**Comment by abiardeau** [comment]
Thanks for your precisions ! I am happy with Jonathan' suggestion of `global_average_sea_level_change_due_to_change_in_ocean_mass` and agree that we can mention barystatic in the definition

**Comment by efisher008** [comment]
These names were accepted as `global_average_steric_sea_level_change` and `global_average_sea_level_change_due_to_change_in_ocean_mass` after discussion in #285 and will be published in v86 of the standard names table. Thank you again for your proposal @abiardeau and @slawchune.
